coderd/autobuild/lifecycle_executor.go

```diff
@@ -331,6 +325,16 @@ func (e *Executor) runOnce(t time.Time) Stats {
 						return xerrors.Errorf("post provisioner job to pubsub: %w", err)
 					}
 				}
+				if dormantNotification != nil {
+					_, err = dormancy.NotifyWorkspaceDormant(
+						e.ctx,
+						e.notificationsEnqueuer,
+						*dormantNotification,
+					)
+					if err != nil {
+						e.log.Warn(e.ctx, "failed to notify of workspace "+dormantNotification.Workspace.Name+" marked as dormant", slog.Error(err))
```

Comment:
Log messages should generally be static, and all variability should be in fields.
I'd also recommend logging the workspace id not name since it's universally unique.

```suggestion
						e.log.Warn(e.ctx, "failed to notify of workspace marked as dormant", slog.Error(err), slog.F("workspace", dormantNotification.Workspace.Id))
```